### PR TITLE
Changed the identifier structure, move to retrieve from TME and added…

### PR DESCRIPTION
**Pull request by nwrigh**
… GTG and build-info

### CODE REVIEW

File: service.go [hunk 1]
```diff
@@ -48,15 +64,26 @@ func (s *subjectServiceImpl) getSubjects() ([]subjectLink, bool) {
 }
 
 func (s *subjectServiceImpl) getSubjectByUUID(uuid string) (subject, bool) {
+	log.Infof("MAP: %v", s.subjectsMap)
```
Comment (tosan88) [inline service.go]: I think this is not necessary

Comment (nwrigh) [inline service.go]: You spotted my debug line opps

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+  && LDFLAGS="-X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+  && cd .. \
   && export GOPATH=/gopath \
   && REPO_PATH="github.com/Financial-Times/subjects-transformer" \
   && mkdir -p $GOPATH/src/${REPO_PATH} \
   && cp -r subjects-transformer/* $GOPATH/src/${REPO_PATH} \
   && cd $GOPATH/src/${REPO_PATH} \
   && go get -t ./... \
-  && go build \
+  && cd $GOPATH/src/${REPO_PATH} \
```
Comment (BerniVarga) [inline Dockerfile]: I think this line isn't needed here...

**Comment by BerniVarga** [comment]
+1

**Pull request merged**